### DISCUSSION TITLE
Jan: Mobile + Sync

### DISCUSSION THREAD

**Discussion by eckartal** [janhq/jan#3626] [Feature Ideas]
## Overview
People want to reach their chats on their phones and continue chatting or create new chats.

## Goals

- Create a mobile app or web interface for accessing and interacting with Jan's chats 
- Implement a secure sync system between the host computer and mobile devices

## Questions

1. How do we manage potential conflicts if changes are made on both mobile and desktop simultaneously?

**Comment by rvrstfn** [comment]
A mobile app would be great.

In case, a mobile inference engine could be integrated, for simple on-device inference of very small models (Qwen 2 1.5B, Gemma 2 2B, at most Phi 3 3B).

This could be coupled with:
- an adapter (LoRA) loader
- some form of lesser model (on device) / better model (online) router
